src/LanguageSearch.php

```diff
@@ -75,6 +79,24 @@ private function getPostLang( $post_args, $post_id ) {
 		return $lang;
 	}
 
+	private function getPostLangAsTranslated( $post_args, $post_id ) {
+		$active_languages = array_keys( $this->sitepress->get_active_languages() );
+		$element_type     = apply_filters( 'wpml_element_type', $post_args["post_type"] );
```

Comment:
@decodekult use single quotes whenever possible

```suggestion
		$element_type     = apply_filters( 'wpml_element_type', $post_args['post_type'] );
```

Reply:
Sure. Thanks!